sqlglot/expressions.py

```diff
@@ -253,6 +254,8 @@ def __deepcopy__(self, memo):
 
         while stack:
             node, copy = stack.pop()
+            copy.arg_key = node.arg_key
```

Comment:
This may be redundant, I believe we set `arg_key` in both `set` and `append` (check `_set_parent`)